tools/IceRpc.Protobuf.Tools/README.md

```diff
@@ -1,2 +1,79 @@
+# Protobuf Tools for IceRPC
 
-TODO
+IceRpc.Protobuf.Tools allows you to compile Protobuf definitions (in `.proto` files) into C# code (in `.cs` files)
+within MSBuild projects.
+
+This package includes the Protobuf compiler, `protoc`, and the `protoc-gen-icerpc-csharp` generator. The `protoc`
+compiler is a native tool with binaries for Linux x64, macOS x64 and Windows (x64). The `protoc-gen-icerpc-csharp`
+generator is a .NET program and requires .NET 8 or later.
+
+Once you've added the IceRpc.Protobuf.Tools NuGet package to your project, the Protobuf files of your project are
+automatically compiled into C# files every time you build this project.
+
+[Source code][source] | [Package][package]
+
+## Protobuf files and Protobuf directories
+
+The Protobuf compiler accepts two main inputs:
+
+- the Protobuf files to compile into C# code (the Protobuf files)
+- directories that contain reference Protobuf files (the Protobuf directories)
```

Comment:
I find this misleading. I'd rather call it the "import path".

A big difference between protoc and slicec is protoc is like a C++ compiler.
slicec compiles all the .slice files it finds in the "directories". protoc does not compile all the .proto files in its import path. The import path is about locating imported files and protoc compiles only source files and the files these source files import.